src/main/java/taxi/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,120 @@
+package taxi.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.ManufacturerDao;
+import taxi.exception.DataProcessingException;
+import taxi.lib.Dao;
+import taxi.model.Manufacturer;
+import taxi.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturer (manufacturer_name, "
+                + "manufacturer_country) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, manufacturer.getModel());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            return manufacturer;
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't insert into DB", ex);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT * FROM manufacturer "
+                + "WHERE manufacturer_id = ? and isDeleted = false ";
```

Comment:
```suggestion
                + "WHERE manufacturer_id = ? and deleted = false ";
```
I'd recommend to not use `method` style names in your column. For now it's a YES from me, don't change this.